### ISSUE TITLE
How I can setup other training parameters while training it on the custom dataaset. ? 

### ISSUE THREAD

**Issue by senhorinfinito** [ultralytics/ultralytics#6615]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I wanted to change some augmentation values, also wanted to use the adamw optimizer  with  label_smoothing  at 0.05 value on a really smaller dataset.  
**Also I wanted change the LR0 and LRF**

How could I setup or make  changes to those parameters ?

**Comment by glenn-jocher** [comment]
@senhorinfinito hello! It's great to see your interest in customizing training parameters for YOLOv8 with your custom dataset. You can adjust training hyperparameters like augmentation values, optimizer type, label smoothing, learning rates and more by modifying the configuration file that pertains to hyperparameters.

You'll find a `hyp.scratch.yaml` file that contains default hyperparameters for training from scratch. You can create a copy of this file and adjust the values according to your needs:

1. For augmentation values, look for keys under the `augment:` section and modify to your preference.
2. To use the AdamW optimizer, change the `optimizer:` field to `adamw`.
3. For label smoothing, you'll find the `label_smoothing:` entry which you can set to `0.05`.
4. Learning rate (`LR0`) and learning rate factor (`LRF`) can be modified under the keys `lr0:` and `lrf:` respectively.

After you've made your changes, you can reference your custom hyperparameter file during training with an argument that points to your updated yaml file. Keep iterating with these settings to find what works best for your dataset.

Feel free to reference the documentation provided on our Docs page for detailed guidance on how to train and tweak YOLOv8 to attain the performance you're aiming for. Happy training! 😊
